### DISCUSSION TITLE
Issue with splash image

### DISCUSSION THREAD

**Discussion by 93lucasp** [capawesome-team/capacitor-mlkit#161] [Q&A]
After that i added the splash image, on Android when i run `startScan()` instead of the camera i see the splash image.
The plugin still works and is able to scan the qr code, but on the screen i don't see the camera but the splash. 
Any idea on this issue?

**Comment by robingenz** [comment]
No such behavior has been reported so far. Feel free create an issue with a [Minimal, Reproducible Example](https://stackoverflow.com/help/minimal-reproducible-example) so I can debug it.
